Data/Passives/MagicPassives.json

```diff
@@ -0,0 +1,53 @@
+﻿[
+	{
+		"internalIdentifier": "Anchor",
+		"referenceId": 1,
+		"maxLevel": 1
+	},
+	{
+		"internalIdentifier": "AddedLife",
+		"referenceId": 2,
```

Comment:
This goes for all referenceId and connections values; is there a reason this is using integers instead of the already existing identifier? Seems redundant.

Reply:
So the reference ID is for the connections themselves, if you use the InternalIdentifier you have no way of differentiating between two of the same passive on the tree. The referenceId is for that specific instance of the passive to be used in the connections array

Reply:
Ah, got it. That's alright then.